qtazu/widgets/login.py

```diff
@@ -87,12 +152,15 @@ def initialize_host(self):
             if gazu_host != "http://gazu.change.serverhost/api":
                 # Assume the host in gazu.client is already set correctly
                 # and copy it into these settings.
-                log.debug("Setting CG-Wire host "
-                          "from gazu.client: %s" % gazu_host)
+                log.debug(
+                    "Setting CG-Wire host " "from gazu.client: %s" % gazu_host
```

Comment:
Your linter/formatter did something weird here. :D It still seems to have the redundant extra parentheses mid-sentence @LedruRollin 

```
"Setting CG-Wire host " "from gazu.client: %s" % gazu_host
```

TO

```
"Setting CG-Wire host from gazu.client: %s" % gazu_host
```